ceph-salt-formula/salt/ceph-salt/cephbootstrap.sls

```diff
@@ -1,5 +1,15 @@
 {% import 'macros.yml' as macros %}
 
+{% set dg_list = pillar['ceph-salt'].get('storage', {'drive_groups': []}).get('drive_groups', []) %}
+
+{% if dg_list | length == 1 %}
+osd crush chooseleaf for single-node cluster:
+  cmd.run:
+    - name: |
+        echo -en "[global]\n        osd crush chooseleaf type = 0\n" > /root/ceph.conf
```

Comment:
why not call `ceph config set ` instead?

Reply:
explained in #100